provisioner/chef-client/provisioner.go

```diff
@@ -193,7 +193,7 @@ func (p *Provisioner) Prepare(raws ...interface{}) error {
 			errs, fmt.Errorf("server_url must be set"))
 	}
 
-	if p.config.SkipInstall == false && p.config.InstallCommand == p.guestOSTypeConfig.installCommand {
+	if !p.config.SkipInstall && p.config.InstallCommand == p.guestOSTypeConfig.installCommand {
```

Comment:
🤔 I think forcing this is a little bit too much and the `== false` way is a little bit more readable and I would leave the choice to programmer/contributor in this case.